app/queries/get_duplicate_candidate_matches.rb

```diff
@@ -0,0 +1,25 @@
+class GetDuplicateCandidateMatches < ApplicationRecord
+  def self.call
+    ActiveRecord::Base.connection.exec_query(
+      "SELECT a.candidate_id, a.first_name, a.last_name, a.postcode, a.date_of_birth, email_address
+    FROM application_forms a
+    JOIN(
+      SELECT application_forms.last_name, application_forms.date_of_birth, application_forms.postcode
+      FROM application_forms
+      WHERE application_forms.previous_application_form_id IS NULL
```

Comment:
I guess we do this to prevent flagging legit subsequent applications as duplicates?

Reply:
Yea exactly as they will always have the same details.

Maybe this should actually just distinct the candidate at the end though? As there's a tiny chance someone could change postcode etc. between applications?

Reply:
Yes — `DISTINCT`ing the candidate feels a bit more intentional? As if to say "we only want application forms created by different candidates providing the same details"